main.py

```diff
@@ -3,10 +3,34 @@
 
 def hello_world():
     print("Hello, World!")
-    sleep_time = random.randint(1, 5)
-    time.sleep(sleep_time)
-    print(f"Slept for {sleep_time} seconds")
+
+
+def fibonacci(n):
```

Comment:
@SeineSailor could you help add unit test function for this `fibonacci(n)` function?